indra/llwindow/llkeyboardsdl.cpp

```diff
@@ -24,7 +24,7 @@
  * $/LicenseInfo$
  */
 
-#if LL_SDL
+#if LL_LINUX
```

Comment:
We used to use SDL for the linux client, however support for linux was allowed to bitrot sometime after 2013.  Now that we're using SDL on the other platforms too this llkeyboardsdl.cpp code should probably be renamed **llkeyboardlinux.cpp**.